greeneeds/users/views.py

```diff
@@ -1,3 +1,56 @@
-from django.shortcuts import render
+import json
+from datetime         import date
 
-# Create your views here.
+from django.views     import View
+from django.http      import JsonResponse
+
+from core.utils       import login_decorator
+from users.models     import Like
+from projects.models  import Project
+
+class LikeView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            data       = json.loads(request.body)
+            user       = request.user
+            project_id = data.get('project_id', None)
+            project    = Project.objects.get(id=project_id)
+
+            if Like.objects.filter(user=user, project=project).exists():
+                Like.objects.filter(user=user, project=project).delete()
+                like_count = Like.objects.filter(project=project).count()
+                return JsonResponse({'MESSAGE': 'SUCCESS', 'LIKE_COUNT':like_count}, status=201)
```

Comment:
좋아요 카운트의 의도가 user의 좋아요 수인가요? project의 좋아요 수 인가요?
```suggestion
                return JsonResponse({'messasge': 'SUCCESS', 'like_count':like_count}, status=201)
```